Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,15 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+const newOrder=order.map(({quantity,itemName,unitPricePence})=>({quantity,itemName,unitPricePence:twoDigitNumber(unitPricePence*quantity)}))
+
+function twoDigitNumber(value){
+  return (value/100).toFixed(2)
+}
+function totalResult(newOrder){
+  return newOrder.reduce((a,{unitPricePence})=>a+Number(unitPricePence),0).toFixed(2)
+
+}
+console.log(newOrder)
```

Comment:
You've done the core parts of this exercise, but the output format is not the same as the example given in the readme - your result is meant to look exactly like the expected result in the README. Please fix this up :)

Reply:
sure thanks! done

Reply:
The format isn't _quite_ the same here - in the README everything is is nice neat columns, but in your output there are different spaces between each column...

Reply:
Yes, I used string and padEnd! :) Thanks for reviewing

Reply:
When I run this now, the items are aligned, but not with the headings:
```
QTY       ITEM                TOTAL 
1     Hot cakes           2.32  
2     Apple Pie           2.78  
1     Egg McMuffin        2.80  
1     Sausage McMuffin    3.00  
2     Hot Coffee          2.00  
4     Hash Brown          1.60  
Total: 14.50
```

How could you write your code differently to make sure the headings and items are aligned? And also to make sure that if you _changed_ the headings (or added more space between them), the items would _also_ stay aligned?